tests/e2e/multi_vc_utils.go

```diff
@@ -0,0 +1,459 @@
+	vmUUID := getNodeUUID(ctx, client, pod.Spec.NodeName)
+	framework.Logf("VMUUID : %s", vmUUID)
+	isDiskAttached, err := multiVCe2eVSphere.verifyVolumeIsAttachedToVMInMultiVC(client, volHandle, vmUUID)
+	if err != nil {
+		return nil, fmt.Errorf("error verifying volume attachment: %v", err)
+	}
+	if !isDiskAttached {
+		return nil, fmt.Errorf("volume is not attached to the node")
+	}
+
+	ginkgo.By("Verify the volume is accessible and filesystem type is as expected")
+	_, err = framework.LookForStringInPodExec(namespace, pod.Name,
+		[]string{"/bin/cat", "/mnt/volume1/fstype"}, "", time.Minute)
+	if err != nil {
+		return nil, fmt.Errorf("error verifying volume accessibility and filesystem type: %v", err)
+	}
+
+	ginkgo.By("Waiting for file system resize to finish")
+	pvclaim, err = waitForFSResize(pvclaim, client)
+	if err != nil {
+		return nil, fmt.Errorf("error waiting for file system resize: %v", err)
+	}
+
+	pvcConditions := pvclaim.Status.Conditions
+	if len(pvcConditions) != 0 {
+		return nil, fmt.Errorf("PVC has conditions")
+	}
+
+	var fsSize int64
+
+	ginkgo.By("Verify filesystem size for mount point /mnt/volume1")
+	fsSize, err = getFSSizeMb(f, pod)
+	if err != nil {
+		return nil, fmt.Errorf("error getting filesystem size: %v", err)
+	}
+	framework.Logf("File system size after expansion: %s", fsSize)
+
+	// Filesystem size may be smaller than the size of the block volume
+	// so here we are checking if the new filesystem size is greater than
+	// the original volume size as the filesystem is formatted for the
+	// first time after pod creation
+	if fsSize < diskSizeInMb {
+		return nil, fmt.Errorf("error updating filesystem size for %q. Resulting filesystem size is %d", pvclaim.Name, fsSize)
+	}
+
+	ginkgo.By("File system resize finished successfully")
+
+	return pod, nil
+}
+
+// govc login cmd
+func govcLoginCmdForMultiVC() string {
+	// configUser := strings.Split(multiVCe2eVSphere.multivcConfig.Global.User[0], ",")
+	// configPwd := strings.Split(multiVCe2eVSphere.multivcConfig.Global.Password[0], ",")
+	// configvCenterHostname := strings.Split(multiVCe2eVSphere.multivcConfig.Global.VCenterHostname[0], ",")
+	// configvCenterPort := strings.Split(multiVCe2eVSphere.multivcConfig.Global.VCenterPort[0], ",")
+	configUser := strings.Split(multiVCe2eVSphere.multivcConfig.Global.User, ",")
+	configPwd := strings.Split(multiVCe2eVSphere.multivcConfig.Global.Password, ",")
+	configvCenterHostname := strings.Split(multiVCe2eVSphere.multivcConfig.Global.VCenterHostname, ",")
+	configvCenterPort := strings.Split(multiVCe2eVSphere.multivcConfig.Global.VCenterPort, ",")
+
+	loginCmd := "export GOVC_INSECURE=1;"
+	loginCmd += fmt.Sprintf("export GOVC_URL='https://%s:%s@%s:%s';",
+		configUser[0], configPwd[0], configvCenterHostname[0], configvCenterPort[0])
+	return loginCmd
+}
+
+func deleteStorageProfile(masterIp string, sshClientConfig *ssh.ClientConfig, storagePolicyName string) error {
+	removeStoragePolicy := govcLoginCmdForMultiVC() +
```

Comment:
Can we use deleteStoragePolicy instead of this method?

Reply:
sure, ack

Reply:
deleteStoragePolicy is expecting profileID *pbmtypes.PbmProfileId of this type, but here in multi-vc the profile id is of string type

tried converting string to this type *pbmtypes.PbmProfileId, but it will require some code change in deleteStoragePolicy method